### PR TITLE
Integrate coreNLP

### PR DESCRIPTION
**Pull request by pbxed**
integrate coreNLP. Modify to return average sentiment score of all sentences passed and create test.

### CODE REVIEW

File: src/main/java/com/codingnomads/betty/logic/services/SentimentAnalyserService.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+            Tree tree = sentence.get(SentimentCoreAnnotations.SentimentAnnotatedTree.class);
+            sentimentResult.setSentimentType(sentence.get(SentimentCoreAnnotations.SentimentClass.class));
+            sentimentResult.setSentimentClass(getClassification(RNNCoreAnnotations.getPredictions(tree)));
+            sentimentResult.setSentimentScore(RNNCoreAnnotations.getPredictedClass(tree));
+            sentimentResultList.add(sentimentResult);
+
+            printSentimentResultBySentence(sentimentResult, sentence);
+        }
+        //TODO: Returning the last result instead of a summary?!?!
```
Comment (snaket2003) [inline src/main/java/com/codingnomads/betty/logic/services/SentimentAnalyserService.java]: Remove comment :)

File: src/main/java/com/codingnomads/betty/presentation/controller/PresentationController.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+
+import java.sql.SQLOutput;
+import java.util.List;
+
+@Controller
+public class PresentationController {
+
+    private final String text = "'Super amazing.’ Maurizio Sarri assesses our chances in the Europa League and the Premier League top-four race... #CHEBUR";
```
Comment (snaket2003) [inline src/main/java/com/codingnomads/betty/presentation/controller/PresentationController.java]: Hey man. Not sure we need a Console presentation controller at all here so, you could just delete it as long as the service works. You could just use tests to make sure it works. Also, the input of the Sentiment Analyzer should be a POJO and not just a String. We could call the service by a performing an action in the front-end

File: src/main/java/com/codingnomads/betty/logic/services/SentimentAnalyserService.java [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+    @Autowired
+    @Qualifier("sentimentProperties")
+    private Properties properties;
+    @Autowired
+    private StanfordCoreNLP pipeline;
+
+    public List<SentimentResult> sentimentResultList = new ArrayList<>();
+
+    public SentimentResult getSentimentResult(String text) {
```
Comment (snaket2003) [inline src/main/java/com/codingnomads/betty/logic/services/SentimentAnalyserService.java]: This service should return a TeamSentimentScore which is the POJO the other service takes in. Not a SentimentResult (you could just delete this POJO completely)

File: src/main/java/com/codingnomads/betty/logic/models/SentimentResult.java [hunk 4]
```diff
@@ -0,0 +1,41 @@
+package com.codingnomads.betty.logic.models;
+
+public class SentimentResult {
```
Comment (snaket2003) [inline src/main/java/com/codingnomads/betty/logic/models/SentimentResult.java]: You can just delete this POJO and use the TeamSentimentScore one. Remember, you also need to pass other information from the tweet / match as per the fields

File: src/main/java/com/codingnomads/betty/BettyApplication.java [hunk 5]
```diff
@@ -1,14 +1,26 @@
...
+	private PresentationController presentationController;
 
 	public static void main(String[] args) {
 		SpringApplication.run(BettyApplication.class, args);
 	}
 
+	@Override
+	public void run(String... args) throws Exception {
+
+		presentationController.start();
```
Comment (snaket2003) [inline src/main/java/com/codingnomads/betty/BettyApplication.java]: No need for a console presentation in our app :)

**Comment by snaket2003** [comment]
Also, you will need to rebase with current master. Probably best to do that and resolve any potential conflicts before making all the other changes

File: src/main/java/com/codingnomads/betty/presentation/controller/PresentationController.java (hunk 2 above)
Comment (xa2113) [reply src/main/java/com/codingnomads/betty/presentation/controller/PresentationController.java]: Hey Jordan, we discussed in the standup today that I'll be doing the integration of the different parts of code (coreNLP, twitter API) into the app and creating the front-end. I can address Cristian's comment here as part of that task. Is this ok with you?

File: src/main/java/com/codingnomads/betty/logic/services/SentimentAnalyserService.java (hunk 3 above)
Comment (xa2113) [reply src/main/java/com/codingnomads/betty/logic/services/SentimentAnalyserService.java]: Same as my comment above.

File: src/main/java/com/codingnomads/betty/logic/models/SentimentResult.java (hunk 4 above)
Comment (xa2113) [reply src/main/java/com/codingnomads/betty/logic/models/SentimentResult.java]: Same as my comment above.

File: src/main/java/com/codingnomads/betty/logic/services/SentimentAnalyserService.java (hunk 1 above)
Comment (xa2113) [reply src/main/java/com/codingnomads/betty/logic/services/SentimentAnalyserService.java]: Do you still need this comment? I can delete it if you dont

**Pull request merged**